README.md

```diff
@@ -40,6 +40,10 @@ HLS stream, code against the regular HTML5 video APIs, and create a
 fast, high-quality video experience across all the big web device
 categories.
 
+Check out the [full documentation](docs/) for details on how HLS works
```

Comment:
Given that we only have a few articles, I feel like that ToC should live in the README.